### DISCUSSION TITLE
Specify the font type for the `text` key in `Hlines` and `Vlines`

### DISCUSSION THREAD

**Discussion by wwojciech** [lvjr/tabularray#484] [Ideas]
It would be great if it is possible to specify the font type for the `text` key in `Hlines` and `Vlines`. I think this is currently not possible and below won't work.
```
vlines = {text={\fontfamily{pcr}\selectfont :}},
```
Of course this one works well:
```
vlines = {text=:},
```

**Comment by muzimuzhi** [comment]
Somehow the value passed to `text` must survive a fully expansion.

```tex
\documentclass{article}
\usepackage{tabularray}

\begin{document}
% or \NewDocumentCommand
\protected\def\myVlineTextStyle{\fontfamily{pcr}\selectfont}

\begin{tblr}{hlines, vlines = {text={\myVlineTextStyle a:}}}
  & & \\
  \\
  \\
\end{tblr}
\end{document}
```

![image](https://github.com/lvjr/tabularray/assets/6376638/0c5503a1-6c39-46b4-b698-4dd7b1e1f19b)

**Reply by wwojciech** [reply]
This works great! Thank you @muzimuzhi!

**Comment by muzimuzhi** [comment]
Ah, it's a known bug. See https://github.com/lvjr/tabularray/issues/303.

I'm closing this discussion and will post my findings there.
